src/test/java/unit/GcpNotebookControlled.java

```diff
@@ -194,7 +195,8 @@ void overrideLocationAndInstanceId() throws IOException {
     // new key-value pair will be appended, existing key-value pair will be updated.
     String newName = "NewOverrideLocationAndInstanceId";
     String newDescription = "\"new override default location and instance id\"";
-    String newMetadata = "NewMetadata1=metadata1,NewMetadata2=metadata2";
+    String newMetadata1 = "NewMetadata1=metadata1";
```

Comment:
I'd declare `String newKey1 = "NewMetadata1;`, etc, and then do `String newEntry1 = newKey1 + "=" + newValue1;`. Then when you go to do your assertions, you don't have to split to get the expected value for `newValue1`.

Reply:
I was also thinking that way to separate the key and value. but the concern will be we should have 4 more variables... I am not sure comparing the more redundant variables and a split sentence, which one will make the code better.